### ISSUE TITLE
Error when running latest version of Next.js

### ISSUE THREAD

**Issue by chimon2000** [react-cosmos/react-cosmos#1639]
### What's wrong?

After upgrading Next.js to the latest, client components cause a `Unhandled Runtime Error` exception.

```
Error: Could not find the module "/Users/edge/Projects/git/nextjs-example/src/components/react-tweet (client).fixture.tsx#default#undefined" in the React Client Manifest. This is probably a bug in the React Server Components bundler.
```

### Steps to reproduce

1.  Update Next.js example
2.  Run `yarn dev` && `yarn cosmos`
3.  Click on the client fixture

### Screenshots

<img width="1371" alt="image" src="https://github.com/react-cosmos/react-cosmos/assets/6907797/414034e1-e152-4165-bd58-41a50ec3bf3e">

### Useful info

- Cosmos version.
- Cosmos config.
- Server terminal output.
- Browser console output.

### Additional context

```
    "@types/node": "20.1.0",
    "@types/react": "18.2.6",
    "@types/react-dom": "18.2.4",
    "next": "^14.2.3",
    "react": "^18.3.1",
    "react-cosmos": "^6.1.1",
    "react-cosmos-next": "^6.1.1",
    "react-cosmos-plugin-open-fixture": "^6.1.1",
    "react-dom": "^18.3.1",
    "react-tweet": "^3.1.1",
    "typescript": "5.0.4"
```

Add any other context about the problem.

**Comment by ovidiuch** [comment]
Hi @chimon2000, thanks for reporting this. Just tried it out and I can reproduce the error.

I've created a PR in the Next.js example repo where I tried to upgrade all dependencies: https://github.com/react-cosmos/nextjs-example/pull/6

The error starts occurring when upgrade next from `14.1.4` to `14.2.0`. Also it's only the client fixture that is broken after the upgrade. The server fixtures continue to work.

I don't know what cases this error at the moment. I don't even no what the "React Client Manifest" is to be frank. I'll do more digging when I find some time but if anyone else with Next.js / RSC experience can lend a hand it would be greatly appreciated.

The `react-cosmos-next` package is a thin wrapper that can be easily debugged. A good starting point would be `node_modules/react-cosmos-next/dist/nextCosmosStaticParams.js`.

**Comment by Fridus** [comment]
Hi,

The error `Could not find the module "...fixture.tsx#default#undefined" in the React Client Manifest. This is probably a bug in the React Server Components bundler.` is present when:
- you use `use client` in fixture 
- since next `14.2.x`

**Comment by Samwakelam** [comment]
I have this issue even with 14.1.4

**Comment by ovidiuch** [comment]
Just merged a fix for this in #1641 

Please try out the canary version until we publish an official release: `react-cosmos@6.1.2-canary.fbe721b.0`. Let me know if you can confirm this works for you, too.

**Comment by Fridus** [comment]
Hi,

```json    
    "next": "^14.2.4",
    "react-cosmos": "^6.1.2-canary.bdbf4f7.0",
    "react-cosmos-next": "^6.1.2-canary.bdbf4f7.0",
    "react-cosmos-plugin-boolean-input": "^6.1.1",
```

@ovidiuch It works for me !

**Comment by rik-iso** [comment]
Also was hitting this issue with the same error - except pointing to one of my cosmos fixtures. Only affecting client component fixtures. Can confirm the canary version fixed it. Thanks!!

```
Error: Could not find the module "/vercel/path0/<pathToMyFixture>.tsx#default#undefined" in the React Client Manifest. This is probably a bug in the React Server Components bundler.
    at eb (/var/task/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:136703)
    at ek (/var/task/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:140326)
    at Array.toJSON (/var/task/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:135755)
    at stringify (<anonymous>)
    at eR (/var/task/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:142219)
    at eE (/var/task/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:142698)
    at Timeout._onTimeout (/var/task/node_modules/next/dist/compiled/next-server/app-page.runtime.prod.js:12:135475)
    at listOnTimeout (node:internal/timers:581:17)
    at process.processTimers (node:internal/timers:519:7) {
  digest: '1047234445'
}
```

**Comment by ovidiuch** [comment]
This issue is fixed in the [v6.2.0 release](https://github.com/react-cosmos/react-cosmos/releases/tag/v6.2.0).
